src/slacko.ml

```diff
@@ -335,6 +335,64 @@ type file_obj = {
   num_stars: int option [@default None];
 } [@@deriving of_yojson { strict = false }]
 
+type field_obj = {
+  title: string option [@default None];
+  value: string option [@default None];
+  short: bool option [@default None];
+} [@@deriving yojson { strict = false }]
+
+let field ?title ?value ?short () = {
+  title;
+  value;
+  short;
+}
+
+type attachment_obj = {
```

Comment:
If `short` is not specified, maybe we can make it default to `false`? Unfortunately the Slack API docs are unclear on what it defaults to or whether it is a tri-state boolean:

> An optional flag indicating whether the value is short enough to be displayed side-by-side with other values.